run-test

```diff
@@ -3,8 +3,8 @@
 rm -rf temp 
 ocropus-nlbin tests/testpage.png -o temp
 ocropus-gpageseg 'temp/????.bin.png'
-ocropus-rpred -n 'temp/????/??????.bin.png'
-ocropus-hocr 'temp/????.bin.png' -o temp.html
```

Comment:
I suggest to change the `run-test` only in the PR #145. And I agree on

> This could be also be copied to a CI-specific version of the test. 

Yes, we should create a separate version of this script for CI.